src/status_im/contexts/onboarding/events.cljs

```diff
@@ -136,42 +136,41 @@
                  [:open-modal :screen/onboarding.enable-biometrics]
                  [:open-modal :screen/onboarding.enable-notifications])}))
 
-(rf/defn seed-phrase-validated
-  {:events [:onboarding/seed-phrase-validated]}
-  [{:keys [db]} seed-phrase key-uid]
-  (let [syncing-account-recovered? (and (seq (:syncing/key-uid db))
-                                        (= (:syncing/key-uid db) key-uid))
-        next-screen                (if syncing-account-recovered?
-                                     :screen/onboarding.create-profile-password
-                                     :screen/onboarding.create-profile)]
-    (if (contains? (:profile/profiles-overview db) key-uid)
-      {:effects.utils/show-confirmation
-       {:title               (i18n/label :t/multiaccount-exists-title)
-        :content             (i18n/label :t/multiaccount-exists-content)
-        :confirm-button-text (i18n/label :t/unlock)
-        :on-accept           (fn []
-                               (re-frame/dispatch [:pop-to-root :screen/profile.profiles])
-                               (re-frame/dispatch
-                                [:profile/profile-selected key-uid]))
-        :on-cancel           #(re-frame/dispatch [:pop-to-root :multiaccounts])}}
-      {:db (-> db
-               (assoc-in [:onboarding/profile :seed-phrase] seed-phrase)
-               (assoc-in [:onboarding/profile :key-uid] key-uid)
-               (assoc-in [:onboarding/profile :color] constants/profile-default-color))
-       :fx [[:dispatch
-             [:navigate-to-within-stack
-              [next-screen
-               (get db
-                    :onboarding/navigated-to-enter-seed-phrase-from-screen
-                    :screen/onboarding.new-to-status)]]]]})))
-
-(rf/defn navigate-to-create-profile
-  {:events [:onboarding/navigate-to-create-profile]}
-  [{:keys [db]}]
-  ;; Restart the flow
-  {:db       (dissoc db :onboarding/profile)
-   :dispatch [:navigate-to-within-stack
-              [:screen/onboarding.create-profile :screen/onboarding.new-to-status]]})
+(rf/reg-event-fx
+ :onboarding/seed-phrase-validated
+ (fn [{:keys [db]} [seed-phrase key-uid]]
+   (let [syncing-account-recovered? (and (seq (:syncing/key-uid db))
+                                         (= (:syncing/key-uid db) key-uid))
```

Comment:
This piece of code already existed. I haven't completely checked, but as far as I did, seems it's impossible that this var is `true`, since we don't execute the flow to validate a seed phrase when we are syncing.

WDYT guys?